environments/india/postgresql.yml

```diff
@@ -32,6 +32,10 @@ dbs:
     - django_alias: icds-ucr
       name: icds-ucr
       django_migrate: True  # ICDS dashboard models are stored here
+    - django_alias: icds-ucr-citus
```

Comment:
Might have approved too soon. Should this get added to `REPORTING_DATABASES` so UCR engine_ids get set?

Reply:
b500712bd